### DISCUSSION TITLE
Why do I keep seeing this person’s messages?

### DISCUSSION THREAD

**Discussion by huangdijia** [community/community#136923] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![QQ_1724747244906](https://github.com/user-attachments/assets/775da782-906c-485e-a4a3-c1a16a1df7fb)

![QQ_1724747518775](https://github.com/user-attachments/assets/20e93425-2758-4dd7-b433-800eb45f948f)

**Comment by mrjosh** [comment, marked as answer]
This guy just wants to notice you so maybe you'll follow him? (look the his followers count, 4.4k)
He just keeps `unfollow/follow` you to keep your attention (maybe with a bot or script) 
I think if you're annoyed by this, you can just block the user to stop the spam?

**Reply by huangdijia** [reply]
How do I block this person?

**Reply by mrjosh** [reply]
If you go his profile page, You can see there's a `Block and report` button at the left bottom section of the about section

**Reply by huangdijia** [reply]
thanks

**Reply by mrjosh** [reply]
![Screenshot 2024-08-27 at 12 05 11 PM](https://github.com/user-attachments/assets/35ba0405-e957-4095-913d-a388a1630ebe)
